badge.go

```diff
@@ -96,28 +98,51 @@ func myNameIs(name string) {
 	display.FillRectangle(0, HEIGHT-2*r-1, WIDTH, r, colors[RED])
 
 	// top text : my NAME is
-	w32, _ := tinyfont.LineWidth(&freesans.Regular12pt7b, "my NAME is")
-	tinyfont.WriteLine(&display, &freesans.Regular12pt7b, (WIDTH-int16(w32))/2, 24, "my NAME is", colors[WHITE])
+	w32, _ := tinyfont.LineWidth(&freesans.Bold12pt7b, "HELLO")
+
+	// top band white
+	display.FillRectangle(30, 4, int16(w32+20), 25, colors[WHITE])
+
+	// my name is text
+	tinyfont.WriteLine(&display, &freesans.Bold12pt7b, (WIDTH-int16(w32))/2, 24, "HELLO", colors[BLACK])
 
 	// middle text
 	w32, _ = tinyfont.LineWidth(&freesans.Bold9pt7b, name)
 	tinyfont.WriteLine(&display, &freesans.Bold9pt7b, (WIDTH-int16(w32))/2, 72, name, colors[BLACK])
 
 	// gophers
-	tinyfont.WriteLineColors(&display, &gophers.Regular32pt, WIDTH-48, 110, "BE", []color.RGBA{getRainbowRGB(100), getRainbowRGB(200)})
+	//tinyfont.WriteLine(&display, &gophers.Regular32pt, WIDTH-48, 110, "BE", []color.RGBA{getRainbowRGB(100), getRainbowRGB(200)})
 }
 
 func myNameIsRainbow(name string) {
 	myNameIs(name)
+	gophersAvail := []string{"A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P"}
+	oldGophers := ""
 
-	w32, _ := tinyfont.LineWidth(&freesans.Bold9pt7b, name)
-	for i := 0; i < 230; i++ {
-		tinyfont.WriteLineColors(&display, &freesans.Bold9pt7b, (WIDTH-int16(w32))/2, 72, name, rainbow[i:])
+	w32, _ := tinyfont.LineWidth(&freesans.Bold12pt7b, name)
+	old_gopher_x := int16(0)
+	for {
+		// pick gophers for animation
+		gopher1 := gophersAvail[rand.Intn(len(gophersAvail))]
+		gopher2 := gophersAvail[rand.Intn(len(gophersAvail))]
+		selectedGophers := fmt.Sprintf("%s%s", gopher1, gopher2)
```

Comment:
Hello Sean, I know this PR is closed, but as a general note, using `fmt` package in TinyGo is not a good idea, it's a somehow big package and should be avoided when possible.